Jenkinsfile

```diff
@@ -251,16 +275,19 @@ pipeline {
                 stage('Used Required Git Hooks') {
                     steps {
                         catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS',
-                                   message: "PR did not get committed with required git hooks.  Please see utils/githooks/README.md.") {
+                                   message: 'PR did not get committed with required git hooks.  ' +
+                                            'Please see utils/githooks/README.md.') {
```

Comment:
Maybe add a function to the start of the Jenkinsfile that returns this text as a string to reduce duplication?